packages/host/app/components/operator-mode/profile-info-popover.gts

```diff
@@ -156,6 +154,73 @@ export default class ProfileInfoPopover extends Component<Signature> {
         </BoxelButton>
       </div>
     {{/if}}
+
+    {{#if this.settingsIsOpenFIXME}}
+      <ModalContainer
+        @onClose={{this.closeSettings}}
+        @title='Settings'
+        @size='large'
+        @centered={{true}}
+        @isOpen={{this.settingsIsOpenFIXME}}
+        data-test-settings-modal
+      >
+        <:sidebar>
+          <Profile />
+        </:sidebar>
+        <:content>
+          FIXME here is settings
+        </:content>
+      </ModalContainer>
+    {{/if}}
+  </template>
+}
+
+export class Profile extends Component<Signature> {
+  @service declare matrixService: MatrixService;
+
+  <template>
+    <div class='profile-popover-body' data-test-profile-icon-container>
+      <ProfileAvatarIcon
+        @userId={{this.matrixService.userId}}
+        class='profile-icon--big'
+      />
+
+      <div class='display-name' data-test-profile-display-name>
+        {{this.matrixService.profile.displayName}}
+      </div>
+
+      <div class='profile-handle' data-test-profile-icon-handle>
+        {{this.matrixService.userId}}
+      </div>
+    </div>
+    <style>
+      .profile-popover-body {
+        margin: auto;
+        display: flex;
+        flex-direction: column;
+      }
+
+      .profile-popover-body > * {
+        margin: auto;
+      }
+
+      .profile-popover-body > .profile-icon {
+        width: 70px;
+        height: 70px;
+        font-size: var(--boxel-font-size-xxl);
+      }
+
+      .display-name {
+        margin-top: var(--boxel-sp-xxxs);
+        font-size: var(--boxel-font-size);
+        font-weight: bold;
+      }
+
+      .profile-handle {
+        margin-top: var(--boxel-sp-xxxxs);
+        color: var(--boxel-500);
+      }
+    </style>
```

Comment:
This can probably be extracted into its own component, it’s the combination of the username icon, display name, and username that show in the popover and setting sidebare.